settings.yml

```diff
@@ -8,7 +8,7 @@ logged_in_url:
   type: string
 
 logged_in_icon:
-  default: "cog"
+  default: "gear"
```

Comment:
does it need a migration?

Reply:
Not for this cog -> gear change - if the setting hasn't been changed, the new default should automatically take effect.

More generally... we could add a migration. But we're only aware of 1 site actually using it, so I don't think it's worth the effort